src/elements/content-preview/__tests__/ContentPreview.test.js

```diff
@@ -955,6 +955,19 @@ describe('elements/content-preview/ContentPreview', () => {
             });
             expect(instance.updatePreviewToken).toBeCalledTimes(1);
         });
+
+        test('should update experiences in preview when previewExperiences changes', async () => {
+            // wrapper.setState({ file });
+            instance.preview = new global.Box.Preview();
+            wrapper.setProps({
+                previewExperiences: {},
+            });
+
+            // TODO remove if statement once https://github.com/box/box-content-preview/pull/1364 is merged
+            if (instance.preview.updateExperiences) {
```

Comment:
I will make a JIRA ticket to update this once `updateExperiences` is defined

Reply:
It looks like Preview is mocked above https://github.com/box/box-ui-elements/blob/master/src/elements/content-preview/__tests__/ContentPreview.test.js#L22, maybe we can update that and do away with this if statement?

Reply:
Great, thank you!